packages/cli/src/templates/project/config-ts.ts

```diff
@@ -4,6 +4,6 @@ import { Provider } from '{{{providerPackageName}}}'
 
 Booster.configure('production', (config: BoosterConfig): void => {
   config.appName = '{{{ projectName }}}'
-  config.provider = Provider
+  config.provider = Provider()
```

Comment:
I was forced to make all providers export a function to avoid breaking the project generator. Having a heterogeneous interface like the one I proposed makes code generation way harder.